src/components/ShareTrajectoryButton/style.css

```diff
@@ -2,11 +2,20 @@
     color: var(--dark-theme-primary-color);
     background-color: var(--dark-theme-header-bg);
     border: none;
+    padding-right: 0px;
+    padding-left: 8px;
 }
 
-.container :global(.ant-btn:hover){
-    color: var(--dark-theme-primary-color);
-    background-color: var(--dark-theme-header-bg);
+.container :global(.ant-tooltip-open.ant-btn) {
+    color: var(--baby-purple-two);
+}
+
+.container .disabled :global(.ant-btn){
+    color: var(--dim-gray-two);
+}
+
+.container :global(.anticon) {
+    font-size: 16px;
```

Comment:
since these are shared across the two components I would put these higher. You may be able to change the disabled color in `ant-vars.less` (you have to restart the server to see the changes). but make sure the other disabled buttons in the light modal are still the way they should be. 

For the other two colors: are all our tooltips this color? We have a number of colors in the `colors.css` file called `tooltip` something, are those not this color? If these are different colors than the other tooltips I would add a classname that indicates that and move this styling up so it applies to both buttons, like in the AppHeader. Same with the anticon thing: is this a more general style across the header? again could use it's own classname if not